src/hist/intervals.py

```diff
@@ -0,0 +1,139 @@
+from typing import Any, Optional
+
+import numpy as np
+
+from .typing import Literal
+
+try:
+    from scipy import stats
+except ImportError:
```

Comment:
```suggestion
except ModuleNotFoundError:
```

Reply:
Do you want me to grab that in `hist.plot` as well? 

https://github.com/scikit-hep/hist/blob/929704218bc9fb24875eafaa1296ab0763e33d72/src/hist/plot.py#L235

I originally had it as `ModuleNotFoundError` but then changed to match.

Reply:
Sure. I thought you had, was surprised to see it revert. `ModuleNotFoundError` is the more precise (Python 3 only) error to catch.